packages/api/src/config.js

```diff
@@ -2,49 +2,31 @@ exports.products = {
   prod_0: {
     order: 0,
     name: "Hacker",
-    lookupKeys: ["price_0"],
-    usage: [
-      {
-        name: "Transcoding",
-        description: "Transcoding (minutes)",
-        price: 0.0,
-      },
-    ],
+    lookupKeys: ["hacker_price"],
+    usage: [],
   },
   prod_1: {
     order: 1,
     name: "Growth",
-    lookupKeys: ["price_1"],
-    usage: [
-      {
-        name: "Transcoding",
-        description: "Transcoding (minutes)",
-        price: 0.005,
-      },
-    ],
+    lookupKeys: ["growth_price"],
+    usage: [],
   },
   prod_2: {
     order: 2,
     name: "Scale",
-    lookupKeys: ["price_2"],
-    usage: [
-      {
-        name: "Transcoding",
-        description: "Transcoding (minutes)",
-        price: 0.005,
-      },
-    ],
+    lookupKeys: ["scale_price"],
+    usage: [],
   },
-  prod_3: {
+  pay_as_you_go_1: {
     order: 3,
     name: "Pay-As-You-Go",
-    lookupKeys: ["price_3"],
+    lookupKeys: ["transcoding_usage", "tstreaming_usage", "tstorage_usage"],
```

Comment:
Nice. Is this a typo? Should be `streaming_usage` and `storage_usage`?

Reply:
Is the reason you named this product `pay_as_you_go_1` and not `pay_as_you_go` because there might be multiple pay-as-you-go products?

Reply:
Yes, it's a typo when I created the prices, but considering it's staging I left it there. Lookup keys needs to be univoque anyway, so when I create the prod products I will make the right things!

About the name, yeah I used the index in case they are going to be future products.
For prod, I was going to name it `pay_as_you_go_prod_1` but I am open to rename everything differently if that's not ok